src/test/scala/edu/ie3/simona/config/ConfigFailFastSpec.scala

```diff
@@ -709,7 +710,7 @@ class ConfigFailFastSpec extends UnitSpec with ConfigTestData {
         "throw an exception if an influxDb1x is configured, but not accessible" ignore {
```

Comment:
Should we go for a test of kafka as well?

Reply:
Found a good way for checking at least the kafka broker connection :)

Reply:
Great!